third_party/VENDOR-LICENSE

```diff
@@ -10618,6 +10618,214 @@ Import: knative.dev/serving/vendor/k8s.io/api
 
 
 
+===========================================================
```

Comment:
Is this change intended?

Reply:
whatever dep generated.  bur probably not, i'll remove it.

Reply:
doesn't deploy without it...